evap/evaluation/migrations/0092_add_course.py

```diff
@@ -0,0 +1,60 @@
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0091_rename_course_to_evaluation'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Course',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name_de', models.CharField(max_length=1024, verbose_name='name (german)')),
+                ('name_en', models.CharField(max_length=1024, verbose_name='name (english)')),
+                ('is_graded', models.BooleanField(default=True, verbose_name='is graded')),
+                ('is_private', models.BooleanField(default=False, verbose_name='is private')),
+                ('gets_no_grade_documents', models.BooleanField(default=False, verbose_name='gets no grade documents')),
+                ('last_modified_time', models.DateTimeField(default=django.utils.timezone.now, verbose_name='Last modified')),
+                ('degrees', models.ManyToManyField(related_name='courses', to='evaluation.Degree', verbose_name='degrees')),
+                ('last_modified_user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='courses_last_modified+', to=settings.AUTH_USER_MODEL)),
+                ('semester', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='courses', to='evaluation.Semester', verbose_name='semester')),
+                ('type', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='courses', to='evaluation.CourseType', verbose_name='course type')),
+            ],
+            options={
+                'verbose_name': 'course',
+                'verbose_name_plural': 'courses',
+                'ordering': ('name_de',),
+                # this is required to prevent database errors about already existing relations and will be changed back in migration 0094
+                'db_table': 'evaluation_course_temp',
```

Comment:
can you add in the comment a source that explains this in more detail?

Reply:
i think there was no source that explicitly stated a solution for this problem. i just thought it might be helpful and it turned out to work.
what happens is that in migration 0092 we're getting this error if the table wasn't manually renamed before:
`django.db.utils.ProgrammingError: relation "evaluation_course_semester_id_name_en_9ebe41fb_uniq" already exists`
which is strange because `Course` was renamed to `Evaluation` in 0091 so i assumed everything related in the database would also have been renamed in this step. but adding a new model named `Course` then results in this error. maybe you have some additional ideas.